fv3config/fv3run/_kubernetes.py

```diff
@@ -26,6 +26,8 @@ def run_kubernetes(
     gcp_secret=None,
     image_pull_policy="IfNotPresent",
     job_labels=None,
+    submit=True,
+    **kwargs,
```

Comment:
As we discussed, in the short term the new kwarg should be listed here explicitly. In the longer term we should get a kubernetes submit command that takes in a command string, and get that command string from another function.